erpnext_documentation/www/docs/user/manual/en/setting-up/delete-company-transactions.md

```diff
@@ -0,0 +1,45 @@
+<!-- add-breadcrumbs -->
+# Delete Company Transactions
+
+**You can delete all the records associated with a company and start afresh, while keeping the other master data like Customers, Items, BOMs intact.**
+
+Often, users setup all the master data and then create a few dummy records. Then they want to delete the dummy records and the company and start over again. This can be done in two ways:
+
+## 1. Transaction Deletion Tool
+
+This feature allows you to delete all the records associated with the specified company, except for the ones belonging to the DocTypes listed in the **Excluded DocTypes** table. 
+
+To access Transaction Deletion Tool, go to:
+> Home > Setup > Transaction Deletion Tool
+
+Tread with caution while using this- records once deleted using this can never be recovered. But if you're certain you want to start over, follow these steps:
+
+1. Go to Transaction Deletion Tool
+1. Create a new doc by clicking on New
+1. Enter the name of the Company whose records you wish to delete
+1. Click on Save
+1. Then click on Submit 
```

Comment:
Better to merge these two steps into one with 'Save and Submit'.

Reply:
Done